build.gradle

```diff
@@ -16,7 +16,7 @@ plugins {
 }
 
 wrapper {
```

Comment:
this change is primarliy cosmetic.  The actual values are in the respective wrapper directory files.